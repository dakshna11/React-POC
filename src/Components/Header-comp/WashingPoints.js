import React from 'react';
import { Link } from 'react-router-dom';
import location from '../img/loc-mark.png'

function WashingPoints(props) {
    return (
        <div>
            <div>
                <div className="WP">
                    <div className='WP-container' >
                        <h1 className="WP-text">Washing Points</h1>
                        <p>
                            <Link className="wordsWP" to={'/Washing Points'}>Home </Link>/
                           <Link className="wordsWP" to={'/Washing Points'}> Location</Link>
                        </p>
                    </div>  
                </div>
            </div>
            

            <div className="location">
            <div className="WPcontainer">
                <div className="WProw">
                    <div className="WP-content">
                        <div >
                            <p style={{color:'red',fontSize:'15px',fontWeight:'bold'}}>
                                <span style={{textDecorationLine:'underline'}}>W A S H I N G  </span>  P O I N T S
                            </p>
                            <h2 style={{fontStyle:'normal',fontSize:'40px',fontWeight:'bolder',color:'#202C45'}}>
                                Car Washing & Care Points
                            </h2>
                        </div>
                        <div className="WP-row">
                            <div className="WP-row1">
                            <div >
                                <div className="location-item">
                                    <img src={location} width='35px' height='35px' style={{paddingTop:'10%'}}></img>
                                    <div className="location-text">
                                        <h3>Car Washing Point</h3>
                                        <p>123 Street, New York, USA</p>
                                        <p><strong>Call:</strong>+012 345 6789</p>
                                    </div>
                                </div>
                            </div>
                            <div >
                                <div className="location-item">
                                <img src={location} width='35px' height='35px' style={{paddingTop:'10%'}}></img>
                                    <div className="location-text">
                                        <h3>Car Washing Point</h3>
                                        <p>123 Street, New York, USA</p>
                                        <p><strong>Call:</strong>+012 345 6789</p>
                                    </div>
                                </div>
                            </div>
                            </div>
                            <div className="WP-row2">
                            <div >
                                <div className="location-item">
                                    <img src={location} width='35px' height='35px' style={{paddingTop:'10%'}}></img>
                                    <div className="location-text">
                                        <h3>Car Washing Point</h3>
                                        <p>123 Street, New York, USA</p>
                                        <p><strong>Call:</strong>+012 345 6789</p>
                                    </div>
                                </div>
                            </div>
                            <div >
                                <div className="location-item">
                                    <img src={location} width='35px' height='35px' style={{paddingTop:'10%'}}></img>
                                    <div className="location-text">
                                        <h3>Car Washing Point</h3>
                                        <p>123 Street, New York, USA</p>
                                        <p><strong>Call:</strong>+012 345 6789</p>
                                    </div>
                                </div>
                            </div>
                            </div>
                        </div>
                    </div>
                    <div className="WP-form">
                        <div className="location-form">
                            <h3>Request for a car wash</h3>
                            <form>
                                <div className="control-group">
                                    <input type="text" className="form-control" placeholder="Name" required="required"/>
                                </div>
                                <div className="control-group">
                                    <input type="email" className="form-control" placeholder="Email" required="required" />
                                </div>
                                <div className="control-group">
                                    <textarea className="form-control" placeholder="Description" required="required"></textarea>
                                </div>
                                <div style={{width:''}}>
                                    <button id="btn-WP" data-testid="btn-WP" type="submit" onClick={()=>alert('form submitted')}>Send Request</button>

                                </div>
                            </form>
                        </div>
                    </div>
                </div>
            </div>
        </div>

        </div>
    );
}

export default WashingPoints;