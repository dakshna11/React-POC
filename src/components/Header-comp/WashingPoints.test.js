import React from "react";
import { shallow } from "enzyme";
import WashingPoints from "./WashingPoints";
import { screen,render } from "@testing-library/react";

let wrapped = shallow(<WashingPoints/>)

describe('WashingPoints', ()=>{
    it('should render the WashingPoints Component correctly', ()=>{
        expect(wrapped).toMatchSnapshot()
    })

    test("renders button", ()=>{
        render(<WashingPoints/>)
        expect(screen.queryByTestId('btn-WP')).toBeInTheDocument()
    })
})