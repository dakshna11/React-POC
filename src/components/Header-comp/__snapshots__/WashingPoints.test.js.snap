// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WashingPoints should render the WashingPoints Component correctly 1`] = `ShallowWrapper {}`;
